### PR TITLE
feat: navbar

### PR DESCRIPTION
**Pull request by amirnathani9**
## change logs
-added basic navbar
-added responsive navbar with the hamburger icon.

## Gifs
navbar

## Preview Link
https://deploy-preview-10--nomark-ui.netlify.app/components/navbar/navbar

### CODE REVIEW

File: components/navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    align-items: center;
+    background-color: var(--bg-color-1);
+    box-shadow: 0px 10px 6px -6px var(--primary-color);
+    width: 100%;
+    padding: 0.5rem;
+    top: 0;
+}
+.navbar-logo-container {
+    width: 30%;
+}
```
Comment (mohitdhatrak) [inline components/navbar/navbar.css]: add 1 line space after each css block for better readability

File: components/navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    .navbar-links{
+        display: flex;
+        width: fit-content;
+        flex-direction: column;
+        align-items: flex-start;
+    }
+    .navbar-links a{
+        font-size: 2rem;
+        display: flex;
+        flex-direction: row;
```
Comment (mohitdhatrak) [inline components/navbar/navbar.css:135]: i think this is not needed, as by default direction is row. Only needed for override is that's the case

**Review by mohitdhatrak — COMMENTED** [review]
Well done, code looked clean mostly! Used semantic tags too at places.

File: components/navbar/navbar.css (hunk 1 above)
Comment (amirnathani9) [reply components/navbar/navbar.css]: ok i will do it

File: components/navbar/navbar.html [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+        <!-- navbar -->
+        <nav class="navbar flex items-center justify-between pt-1 section-center z-5">
+            <a class="humburger-icon icon-open font-size-6"><i class="fas fa-bars ml-5"></i></a>
+            <a class="humburger-icon icon-close font-size-6 displayNone"><i class="fas fa-bars ml-5"></i></a>
+            <a href="../../index.html" class="mr-5"><img src="../../assests/logo-primary.png" alt="NOMARK"
+                    class="nomark-logo"></a>
+        </nav>
+        <!-- side-bar -->
+        <div class="sidebar sidebarRemove flex flex-col scroll-style z-4 pr-7 p-2">
+            <h3 class="sidebar-heading font-size-5">Getting Started</h3>
```
Comment (devdeadviz) [inline components/navbar/navbar.html:30]: Use a single type of class name.

**Review by devdeadviz — APPROVED** [review]
Everything looks good to me!

File: components/navbar/navbar.css (hunk 2 above)
Comment (amirnathani9) [reply components/navbar/navbar.css:135]: I used the direction column in normal size and in the media query I m giving the direction row

**Pull request merged**